cnames_active.js

```diff
@@ -915,7 +915,8 @@ var cnames_active = {
   "humble": "humblejs.github.io",
   "hx": "hifocus.github.io/www.hxis.me",
   "hybrids": "hosting.gitbook.com", // noCF
-  "hyde": "gheek.github.io/hyde", // noCF? (don´t add this in a new PR)
+  "hyde": "gheek.github.io/hyde", // noCF? (don´t add this in a new PR),
```

Comment:
```suggestion
  "hyde": "gheek.github.io/hyde", // noCF? (don´t add this in a new PR)
```
Please revert the change to this line